### PR TITLE
Adding assign sensitivity label api documentation

### PR DESCRIPTION
**Pull request by jaLuthra**
Our team is planning to expose a new Graph API which will allow users to assign sensitivity label asynchronously on a given drive item
This PR will document the API.

### CODE REVIEW

File: api-reference/beta/api/driveitem-assignsensitivitylabel.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+| justificationText   | string                            | Optional. Justification text for audit purposes. Required when downgrading/removing label.                          |
+
+### sensitivityLabelAssignmentMethod values
+
+| Member             | Description                                    |
+|:------------------ |:-----------------------------------------------|
+| standard           | The assignment method for the label is standard.|
+| privileged         | The assignment method for the label is privileged. Indicates that the label is applied manually by a user or by an admin.|
+| auto               | Indicates that the label is applied automatically by the system due to a configured policy, such as default label or auto-classification of sensitive content.|
+| unknownFutureValue | Evolvable enumeration sentinel value. Do not use.|
```
Comment (isvargasmsft) [inline api-reference/beta/api/driveitem-assignsensitivitylabel.md:64]: Why are we defining this enumeration here? This enum is already defined in [its parent topic here](https://docs.microsoft.com/en-us/graph/api/resources/sensitivitylabelassignment?view=graph-rest-beta).

File: api-reference/beta/api/driveitem-assignsensitivitylabel.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}. Required.|
+
+## Request body
+| Name                | Value                             |Description                                                                                                          |
+|:--------------------|:----------------------------------|:--------------------------------------------------------------------------------------------------------------------|
+| sensitivityLabelId  | string                            | Required. Id of the sensitivity label to be assigned, or empty string to remove the sensitivity label.              |
+| assignmentMethod    | sensitivityLabelAssignmentMethod  | Optional. The assignment method of the label on the document. Whether the assignment of the label was done automatically, standard or as a privileged operation (The equivalent to an administrator operation).                                                                            |
+| justificationText   | string                            | Optional. Justification text for audit purposes. Required when downgrading/removing label.                          |
```
Comment (isvargasmsft) [inline api-reference/beta/api/driveitem-assignsensitivitylabel.md:55]: Please add an intro sentence to this table. For example: 
```suggestion
In the request body, provide a JSON object with the following parameters.

| Name                | Value                             |Description                                                                                                          |
|:--------------------|:----------------------------------|:--------------------------------------------------------------------------------------------------------------------|
| sensitivityLabelId  | string                            | Required. Id of the sensitivity label to be assigned, or empty string to remove the sensitivity label.              |
| assignmentMethod    | sensitivityLabelAssignmentMethod  | Optional. The assignment method of the label on the document. Whether the assignment of the label was done automatically, standard or as a privileged operation (The equivalent to an administrator operation).                                                                            |
| justificationText   | string                            | Optional. Justification text for audit purposes. Required when downgrading/removing label.                          |
```

File: api-reference/beta/api/driveitem-assignsensitivitylabel.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+| Member             | Description                                    |
+|:------------------ |:-----------------------------------------------|
+| standard           | The assignment method for the label is standard.|
+| privileged         | The assignment method for the label is privileged. Indicates that the label is applied manually by a user or by an admin.|
+| auto               | Indicates that the label is applied automatically by the system due to a configured policy, such as default label or auto-classification of sensitive content.|
+| unknownFutureValue | Evolvable enumeration sentinel value. Do not use.|
+
+## Response
+
+If successful, the API returns details about how to [monitor the progress](/graph/long-running-actions-overview) of the assign sensitivity label operation, upon accepting the request.
```
Comment (isvargasmsft) [inline api-reference/beta/api/driveitem-assignsensitivitylabel.md:68]: Please mention the response code that is returned with the operation, the headers and a clarification that it doesn't return anything in the response body if applicable.

File: api-reference/beta/api/driveitem-assignsensitivitylabel.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+POST /drives/{drive-id}/items/{item-id}/assignSensitivitylabel
+POST /drives/{drive-id}/root:/{item-path}/assignSensitivitylabel
+POST /groups/{group-id}/drive/items/{item-id}/assignSensitivitylabel
+POST /groups/{group-id}/drive/root:/{item-path}/assignSensitivitylabel
+POST /me/drive/items/{item-id}/assignSensitivitylabel
+POST /me/drive/root:/{item-path}/assignSensitivitylabel
+POST /sites/{site-id}/drive/items/{item-id}/assignSensitivitylabel
+POST /sites/{site-id}/drive/root:/{item-path}/assignSensitivitylabel
+POST /users/{user-id}/drive/items/{item-id}/assignSensitivitylabel
+POST /users/{user-id}/drive/root:/{item-path}/assignSensitivitylabel
```
Comment (isvargasmsft) [inline api-reference/beta/api/driveitem-assignsensitivitylabel.md:42]: According to the CSDL PR, the name of the operation has the L capitalized: assignSensitivityLabel.
```suggestion
POST /drives/{drive-id}/items/{item-id}/assignSensitivityLabel
POST /drives/{drive-id}/root:/{item-path}/assignSensitivityLabel
POST /groups/{group-id}/drive/items/{item-id}/assignSensitivityLabel
POST /groups/{group-id}/drive/root:/{item-path}/assignSensitivityLabel
POST /me/drive/items/{item-id}/assignSensitivityLabel
POST /me/drive/root:/{item-path}/assignSensitivityLabel
POST /sites/{site-id}/drive/items/{item-id}/assignSensitivityLabel
POST /sites/{site-id}/drive/root:/{item-path}/assignSensitivityLabel
POST /users/{user-id}/drive/items/{item-id}/assignSensitivityLabel
POST /users/{user-id}/drive/root:/{item-path}/assignSensitivityLabel
```

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @jaLuthra, 

Thank you very much for your work in this PR. Please take a look at a few comments I made.
 
Kind regards, 

Isaac

File: api-reference/beta/api/driveitem-assignsensitivitylabel.md (hunk 4 above)
Comment (jaLuthra) [reply api-reference/beta/api/driveitem-assignsensitivitylabel.md]: fixed

File: api-reference/beta/api/driveitem-assignsensitivitylabel.md (hunk 3 above)
Comment (jaLuthra) [reply api-reference/beta/api/driveitem-assignsensitivitylabel.md]: updated

File: api-reference/beta/api/driveitem-assignsensitivitylabel.md (hunk 2 above)
Comment (jaLuthra) [reply api-reference/beta/api/driveitem-assignsensitivitylabel.md]: updated.

File: api-reference/beta/api/driveitem-assignsensitivitylabel.md (hunk 1 above)
Comment (jaLuthra) [reply api-reference/beta/api/driveitem-assignsensitivitylabel.md]: it's not directly a parent topic, sensitivityLabelAssignment is a resourceType which is part of return response of https://docs.microsoft.com/en-us/graph/api/driveitem-extractsensitivitylabels?view=graph-rest-beta&tabs=http. But in this API sensitivityLabelAssignment is not part of response. But I do get your point of not repeating same definition again. I have updated this with link to enum definition. Please have a look

**Comment by jaLuthra** [comment]
@isvargasmsft 
I have resolved your comments please have a re-look into this.

**Review by isvargasmsft — APPROVED** [review]

**Pull request merged**